<html>
<head>

<meta charset="utf-8" />
<title>iGEM WHU-China</title>
<link href="main.css" rel="stylesheet" type="text/css" />

</head>
<body>

<nav>
  <ul>
    <li class='home'><a href="">Home</a></li>
    <div class='hidden no-exist'></div>
    <li class='project'><a href="">Project</a></li>
    <ul class='hidden'>
      <li><a href="#background">Background</a></li>
      <li><a href="#project1">the Coloration System</a></li>
      <li><a href="#project2">the Street Cleaner System</a></li>
      <li><a href="#project3">the Lysis System</a></li>
      <li><a href="#application">Application</a></li>
    </ul>
    <li class='results'><a href="">Results</a></li>
    <ul class='hidden'>
      <li><a href="#result1">the Coloration System</a></li>
      <li><a href="#result2">the Street Cleaner System</a></li>
      <li><a href="#result3">the Lysis System</a></li>
      <li><a href="#future">Future Work</a></li>
    </ul>
    <li class='modeling current'><a href="">Modeling</a></li>
    <ul class=''>
      <li><a href="#model-objective">Objective</a></li>
      <li><a href="#model-description">Description</a></li>
      <li><a href="#model-result">Result</a></li>
    </ul>
    <li class='achieve'><a href="">Achievements</a></li>
    <ul class='hidden'>
      <li><a href="#judging">Judging Form</a></li>
      <li><a href="#parts">Parts Registered</a></li>
    </ul>
    <li class='practice'><a href="">Human Practice</a></li>
    <ul class='hidden'>
      <li><a href="#research">Research: Formaldehyde in cars</a></li>
      <li><a href="#helping">Helping Other Teams</a></li>
      <li><a href="#popularization">Popularization</a></li>
      <li><a href="#communication">Communication</a></li>
    </ul>
    <li class='team'><a href="">Team</a></li>
    <ul class='hidden'>
      <li><a href="#members">Members</a></li>
      <li><a href="#attribution">Attribution</a></li>
      <li><a href="#gallery">Gallery</a></li>
    </ul>
    <li class='safety'><a href="">Safety</a></li>
    <ul class='hidden'>
      <li><a href="#brief">Brief Answers to Safety Questions</a></li>
      <li><a href="#issues">Safety Issues evaluation</a></li>
    </ul>
    <li class='notebook'><a href="">Notebook</a></li>
    <ul class='hidden'>
      <li><a href="#timeline">Timeline</a></li>
      <li><a href="#methods">Methods</a></li>
    </ul>
  </ul>
</nav>

<div id="banner">
<div class="logo"><img src="main-images/Whu-logo.jpg" /></div>
<div class="project-name">Formaldehyde Terminator</div>
</div>

<main>

<section id="home" class='hidden'>
<div>

<h1>Formaldehyde Terminator</h1>
<div class="logo"><img src="main-images/Whu-logo.jpg" /></div>
<div class="figure"><img src="main-images/flow.png" /></div>

</div>
</section>

<section id="project" class='hidden'>
<div>

<div class="overview">
<p>Formaldehyde is known to be a human carcinogen which do a lot of
harm to human beings, especially whose house is newly decorated. And
to make matters worse, there are no effective and
non-secondary-pollution method to eliminate indoor formaldehyde. To
solve this problem, we developed a method of detection and removal of
indoor formaldehyde.</p>

<p>Using synthetic biology, we developed a method of detection and
removal of indoor formaldehyde, which can solve this problem without
secondary-pollution.</p>
</div>

<div class="background">
<a name="background"></a>
<h2>Background: About Formaldehyde</h2>

<h3>Properties</h3>

<p>Formaldehyde is the simplest aldehyde. It is a highly reactive gas
and is formed by oxidation or incomplete combustion of
hydrocarbons. Formaldehyde exists at room temperature as a nearly
colorless gas with a pungent, suffocating odor. In its pure state,
formaldehyde is not easily handled, because it is extremely reactive
and polymerizes readily. The primary form of formaldehyde in dilute
aqueous solutions is its monomeric hydrate, methylene glycol
(methanediol) and the primary forms in concentrated solutions are
oligomers and polymers of polyoxymethylene glycols.<ref>[1]</ref></p>

<p>The physical and chemical properties of formaldehyde are summarized
in Table 1.</p>

<div class="image"><img src="images/P-B-table.png" />
<p>Table 1. Physical and chemical properties of formaldehyde</p></div>

<p>Formaldehyde is an important precursor to many other materials and
chemical compounds. In view of its widespread use, toxicity and
volatility, exposure to formaldehyde is a significant consideration
for human health. In 2011, the US National Toxicology Program
described formaldehyde as "known to be a human
carcinogen".<ref>[2]</ref></p>

<h3>Applications</h3>

<h4>3.1 Construction and Decorative Products</h4>

<p>Since the invention of the particle board in Germany in the 1940s,
the use of formaldehyde in glues and resins has revolutionised the
construction industry. Formaldehyde-based resins allow wood chips,
sawdust and even some recycled wood waste to be combined to create
wood products that are durable, cost-effective and high performance
alternatives to solid wood. Formaldehyde-based resins also provide
better dimensional stability and mould resistance.</p>

<div class="image"><img src="images/P-B-1.png" />
<p>Figure 1: Formaldehyde in Construction and Decorative Products</p></div>

<h4>Automotive Applications</h4>

<p>Many 'under-the-hood components' such as fuel pumps, transmission
parts and brake pads are produced with formaldehyde-based
resins. Other formaldehyde based applications include the decorative
laminates of car interiors, engine lubricants, vulcanized rubber tyres
and lightweight polyurethane foams for automobile door insulation.</p>

<div class="image"><img src="images/P-B-2.jpg" />
<p>Figure 1: Formaldehyde in Construction and Decorative Products</p></div>

<h3>Harm</h3>

<h4>Toxic effects</h4>

<p>Formaldehyde is a highly reactive chemical that causes tissue
irritation and damage on contact. Formaldehyde concentrations that
have been associated with various toxic effects in humans show wide
interindividual variation and are route dependent. Symptoms are rare
at concentrations below 0.5 ppm; however, upper airway and eye
irritation, changes in odor threshold, and neurophysiological effects
(e.g., insomnia, memory loss, mood alterations, nausea, fatigue) have
been reported at concentrations &le; 0.1 ppm. The most commonly reported
effects include eye, nose, throat, and skin irritation.</p>

<h4>Carcinogenicity</h4>

<p>Formaldehyde is known to be a human carcinogen based on sufficient
evidence of carcinogenicity from studies in humans and supporting
studies on mechanisms of carcinogenesis.</p>

<p>Epidemiological studies have demonstrated a causal relationship
between exposure to formaldehyde and cancer in humans. Causality is
indicated by consistent findings of increased risks of nasopharyngeal
cancer, sinonasal cancer, and myeloid leukemia among individuals with
higher measures of exposure to formaldehyde (exposure level or
duration), which cannot be explained by chance, bias, or
confounding.</p>

<p>Formaldehyde exposure is associated with multiple modes of action
related to carcinogenicity, such as DNA reactivity, gene mutation,
chromosomal breakage, aneuploidy, epigenetic effects (binding to
lysine residues of histones), glutathione depletion, oxidative stress,
and cytotoxicity-induced cellular proliferation (Lu et al. 2008,
Guyton et al. 2009, NTP 2010). There is evidence for a genotoxic mode
of action for formaldehyde-induced cancer; however, the mechanisms by
which formaldehyde causes cancer are not completely understood and
most likely involve several modes of action.<ref>[1]</ref></p>

<div class="image"><img src="images/P-B-3.png" />
<p>Figure 1: Formaldehyde in Construction and Decorative Products</p></div>

<p style="background-color: red">To detect and remove the
formaldehyde, we design and construct three systems to build our
Formaldehyde Terminator.</p>

</div>

<div class="project1">
<a name="project1"></a>
<h2>Coloration system</h2>

<p> Since we would like to use formaldehyde as an input in our
design,can we find or construct a biobrick sensing the presence of
formaldehyde?  In Bacillus subtilis 168, there is an operon called
hxlAB operon.<ref>[1][2]</ref></p>

<div class="image"><img src="images/P-D-1-formaldehyde-promoter.png" />
<p>Figure 1 The formaldehyde promoter shown in the red column</p>
</div>

<p>This operon controls the expression of two key enzymes in the
ribulose monophosphate pathway that are involved in formaldehyde
fixation, 3-hexulose-6-phosphate synthase and
6-phospho-3-hexuloisomerase. <ref>[3]</ref> Expression of the hxlAB
operon is induced by the presence of formaldehyde.  HxlR protein can
enhance the operon to some extent. The hxlR gene that encodes HxlR
protein is located upstream the hxlAB operon.
</p>

<p>We get a part of the operon (which we name it P-formaldehyde,shown
in a red column in Figure 1) by PCR , add a fluorescent protein or
chromoprotein downstream and transform the plasmid(Figure2) into
E.coli DH5a. In this way, we can know whether there is formaldehyde or
not from the color change.</p>

<div class="image"><img src="images/P-D-2-GFP-reporter-added-downstream-of-formaldehyde-promoter.png" />
<p>Figure 2 A GFP reporter is added downstream of the formaldehyde promoter</p>
</div>

</div>

<div class="project2">
<a name="project2"></a>
<h2>The street cleaner system</h2>

<p>PADH, formaldehyde dehydrogenase from Pseudomonas putida is a
formaldehyde dehydrogenase which can oxidize free formaldehyde to
formate independent of GSH. <ref>[4]</ref> FDH, formate dehydrogenase
from Candida boidinii can oxidize formate to carbon dioxide in the
presence of NAD+.<ref>[5][6]</ref></p>

<div class="image"><img src="images/P-D-3--mechanism-of-two-enzymatic-reactions .png" />
<p>Figure 3 The mechanism of two enzymatic reactions</p>
</div>

<p>We would like to use this two enzymes to degrade formaldehyde into
water and carbon dioxide.  In our design,the expression of this two
enzymes can be controlled by the P-formaldehyde so that the presence
of formaldehyde can induce the expression of them. We can also use
constitutive promoter to guarantee the expression level.</p>

<div class="image"><img src="images/P-D-4-two-enzymes-added-downstream-of-formaldehyde-promoter.png" />
<p>Figure 4 Two enzymes are added downstream of the formaldehyde promoter</p>
</div>

</div>

<div class="project3">
<a name="project3"></a>
<h2>Lysis system</h2>

<p>Lysis system has two important roles in our design.  One is about
biosafety; the other is to help the enzymes come out of the bacteria
so that they can work better in the environment.</p>

<p>We use Lux system, E7 protein and our P-formaldehyde to construct
this system.</p>

<p>Lux system is a quorum sensing system , consisting of luxI, luxR
and P-lux. luxI is a synthase for a small molecule called
acyl-homoserine lactone (AHL). AHL can bind to the LuxR protein and
then the composite can stimulate transcription from the right hand lux
promoter P-lux.<ref>[7]</ref></p>

<div class="image"><img src="images/P-D-5-mechanism-of-lux-system.png" />
<p>Figure 5 The mechanism of the lux system</p>
</div>

<p>The E7 lysis protein is a key component of the SOS response system
in E.coli and functions to export bacteriocins into the extracelluar
space under stressful environmental conditions. Besides, this protein
only has 47 amino acids and thus can be easily utilized as an modular
part in a constructed biobrick which helps the expressed foreign
protein easier to be collected or take effect outside
cell.<ref>[8][9]</ref> In this case, E7 lysis protein is used to
release enzyme FDH and enzyme PADH to the extracellular space so that
they can degrade formaldehyde in a more efficient way.We design this
system as following:</p>

<div class="image"><img src="images/P-D-6-system-release-enzymes-when-stimulated-by-formaldehyde.png" />
<p>Figure 6 A system which can release two enzymes out of the cells when stimulated by formaldehyde</p>
</div>

<p>When the P-formaldehyde is activated, the bacteria will express
FDH, PADH and LuxI protein. LuxI synthesis AHL and AHL binds to LuxR
so that the P-lux is activated and the E7 protein is expressed. When
E7 reaches a certain level, the bacteria lyse.</p>

</div>

<div class="application">
<a name="application"></a>
<h2>Application:</h2>

<p>To make our detection and street cleaner systems more convenient
and functional for others to use in both daily life and industry (this
is also a requirement for a good iGEM project), we have made a
device (see "A Device for Daily Life") and got a detailed plan about
industrial application (see "Devices for Industrial
Application").</p>

<h3>A Device for Daily Life</h3>

<p>We have made a convenient device(showed below) to hold our E.coli so that we can detect the formaldehyde gas easier. A electronic formaldehyde detector was added so that we can compare our detection system with it.</p>
 
<div class="line-wrapper">

<div class="image"><img src="images/P-A-1.jpg" /></div>
<div class="image"><img src="images/P-A-2.jpg" /></div>

</div>

<div class="clearfix"></div>

<p>In our future work, we will combine this device with automatic
recording machine. By connecting them, we can get data automatically
and precisely. Then we will optimize it by diminishing its volume and
embellishing its appearance so that it can be easier to use.</p>

<h3>Devices for Industrial Application</h3>

<p>Although there still remains a lot for our project to come to be
used practically, we can hope that one day our dream can come true. As
we know from the information above, our systems can be used
separately; one way is that our Street Cleaner System can help to
clean formaldehyde from industrial effluent. But if we want to use it
in a higher efficient way, we may make use of engineering
machine. Here we show two possible devices:</p>

<p>The first one is Bacterial Board, we can see from DEVICE I.</p>

<div class="image"><img src="images/P-A-3.jpg" /></div>
 
<p>This device is mainly composed by two semi-permeable membranes and
a container that contains bacteria, this device is inclined against
water direction, and this can limit the influence of deposition in
effluent to semi-membrane. Bacteria in container is access to culture
jar, it can keep flowing and provide bacteria with successive nutrient
and oxygen. It can be used to effluent that has low poison. After
setting series of these devices in an area of sewage duct, we can
clean formaldehyde efficiently.</p>

<p>The second one is bacterial windmill, we can see from DEVICE II,</p>

<div class="image"><img src="images/P-A-4.jpg" /></div>
 
<p>The windmill has a lot of containers, these containers contain
bacteria, when the windmill rotates toward water direction, they can
contact effluent one by one, and then it can be above the surface of
effluent, it provides bacteria with chance to get both formaldehyde
and oxygen, bacteria can also avoid being submerged in harmful
effluence for long time, and it can also limit the influence of
deposition in water. This device can be used in high poisonous
effluence.</p>

</div>

<div class="reference">
<h2>reference</h2>

<ul>
<li>
[1] Yurimoto H, Hirai R, Matsuno N, et al. HxlR, a member of the DUF24 protein family, is a DNA&copy; binding protein that acts as a positive regulator of the formaldehyde&copy;inducible hxlAB operon in Bacillus subtilis[J]. Molecular microbiology, 2005, 57(2): 511-519.
</li>
<li>
[2] Huyen N T T, Eiamphungporn W, M&auml;der U, et al. Genome&copy; wide responses to carbonyl electrophiles in Bacillus subtilis: control of the thiol&copy; dependent formaldehyde dehydrogenase AdhA and cysteine proteinase YraA by the MerR&copy; family regulator YraB (AdhR)[J]. Molecular microbiology, 2009, 71(4): 876-894.
</li>
<li>
[3] Yasueda H, Kawahara Y, Sugimoto S. Bacillus subtilis yckG andyckF Encode Two Key Enzymes of the Ribulose Monophosphate Pathway Used by Methylotrophs, and yckH Is Required for Their Expression[J]. Journal of bacteriology, 1999, 181(23): 7154-7160.
</li>
<li>
[4] Ito K, Takahashi M, Yoshimoto T, et al. Cloning and high-level expression of the glutathione-independent formaldehyde dehydrogenase gene from Pseudomonas putida[J]. Journal of bacteriology, 1994, 176(9): 2483-2491.
</li>
<li>
[5] Sheng B, Zheng Z, Lv M, et al. Efficient Production of (R)-2-Hydroxy-4-Phenylbutyric Acid by Using a Coupled Reconstructed d-Lactate Dehydrogenase and Formate Dehydrogenase System[J]. PloS one, 2014, 9(8): e104204.
</li>
<li>
[6] Khangulov S V, Gladyshev V N, Dismukes G C, et al. Selenium-containing formate dehydrogenase H from Escherichia coli: a molybdopterin enzyme that catalyzes formate oxidation without oxygen transfer[J]. Biochemistry, 1998, 37(10): 3518-3528.
</li>
<li>
[7] Winson M K, Camara M, Latifi A, et al. Multiple N-acyl-L-homoserine lactone signal molecules regulate production of virulence determinants and secondary metabolites in Pseudomonas aeruginosa[J]. Proceedings of the National Academy of Sciences, 1995, 92(20): 9427-9431.	
</li>
<li>
[8]Lin L J R, Liao C C, Chen Y R, et al. Induction of membrane permeability in Escherichia coli mediated by lysis protein of the ColE7 operon[J]. FEMS microbiology letters, 2009, 298(1): 85-92.
</li>
<li>
[9] Chak K F, Kuo W S, James R. Cloning and characterization of the ColE7 plasmid[J]. Journal of general microbiology, 1991, 137(1): 91-100.
</li>
</ul>

</div>

</div>
</section>

<section id="results" class='hidden'>
<div>

<div class="result1">
<a name="result1"></a>
<h2>1 Coloration system</h2>

<p>We have successfully amplified the P-formaldehyde gene (595bp) from
Bacillus subtilis 168 using different annealing temperature(60&#8451;, 58&#8451;,
56&#8451; corresponding to products 1, 2, 3 respectively).<ref>[1]</ref></p>

<div class="image"><img src="images/R-001-PCR-product-formaldehyde-promoter.png" /></div>
Figure 1 The PCR product of formaldehyde promoter.

<p>Then we ligate P-formaldehyde with the pSB1C3 backbone using
standard assembly to make
<a href="http://parts.igem.org/Part:BBa_K1334002">BBa_K1334002</a>.</p>

<div class="image"><img src="images/R-002-formaldehyde-promoter-pSB1C3-backbone-BBa_K1334002.png" /></div>
Figure 2 Formaldehyde promoter in the pSB1C3 backbone (BBa_K1334002).

<p>In order to prove the function of P-formaldehyde in the system
described above, we add a GFP reporter gene to the downstream of
it. Thus by detecting relative fluorescence intensity we get to know
whether the P-formaldehyde works.<ref>[1][2][3]</ref></p>

<div class="image"><img src="images/R-003-GFP-reporter-added-downstream-formaldehyde-promoter.png" /></div>
Figure 3 A GFP reporter is added downstream of the formaldehyde promoter

<p>We transform P-formaldehyde+GFP plasmid into E.coli strain
DH5&alpha;. The positive clone is screened out and used as experimental
group. After cultured to OD600 reaching 1.0 (mid-log phase of E.coli
growth curve), formaldehyde is added to broth to 1mM. bacteria samples
are collected at different time and their fluorescence intensity and
absorbance are measured by microplate reader.</p>

<p>From the following two figures, we can see significant increase in
relative fluorescence intensity after formaldehyde stimulation,
indicating that our coloration system works well. In addition,the
growth condition curve shows that the difference of the
fluorescence/absorbance is not caused by the amount of bacteria.</p>

<div class="image"><img src="images/R-004-function-test-formaldehyde-promoter.png" />
<p>Figure 4 Formaldehyde promoter function chart.</p>
</div>

<div class="image"><img src="images/R-005-growth-condition-Ecoli-strain-DH5a-after-formaldehyde-stimulation.png" />
<p>Figure 5 The growth condition of E.coli strain DH5a after formaldehyde stimulation.</p>
</div>

</div>

<div class="result2">
<a name="result2"></a>
<h2>2 Street Cleaner System</h2>

<p>We construct biobricks: Part: BBa_K1334016, BBa_K1334015 and
BBa_K1334017. And we construct a plasmid as following:</p>

<div class="image"><img src="images/R-006-formate-dehydrogenase-generator.png" /></div>
Figure 6 The Formate dehydrogenase generator.

<div class="image"><img src="images/R-007-formaldehyde-dehydrogenase-generator.png" /></div>
Figure 7 The Formaldehyde dehydrogenase generator. 

<p>And we transform the plasmids into the expression strain of
E.coli. we incubate the E.coli and use the bacteria lysate to run a
SDS-PAGE. The FDH from E.coli is about 80kD while we can see a 25kD
fragment and the protein PADH is 42.5kD while there is a fragment
between 15 and 25kD region. So we need to modify our biobrick to
express the two enzymes.<ref>[4][5][6]</ref></p>

<div class="image"><img src="images/R-008-SDS-PAGE-for-FDH-generator.png" /></div>
Figure 8 SDS-PAGE for the FDH generator.

<div class="image"><img src="images/R-009-SDS-PAGE-for-PADH-generator.png" /></div>
Figure 9 SDS-PAGE for the PADH generator.

</div>

<div class="result3">
<a name="result3"></a>
<h2>3 lysis system</h2>

<p>(1)	Lux system</p>

<p>As we have mentioned in our design, we construct a biobrick to test
whether the lux system works or not.</p>

<div class="image"><img src="images/R-010-signal-complex-to-test-lux-system-BBa_K1334020.png" /></div>
Figure 10 A signal complex to test whether the lux system works (<a href="http://parts.igem.org/Part:BBa_K1334020">BBa_K1334020</a>).

<p>We transform this plasmid into the E.coli strain DH5a. After
exogenous AHL (3OC6HSL ,final concentration:100nM)stimulation, we
detect the relative fluorescence intensity of RFP. The following
figures shows that the lux system works well.<ref>[7]</ref></p>

<div class="image"><img src="images/R-011-function-test-BBa_K1334020.png" /></div>
Figure 11 The function test of our part BBa_K1334020 in which the relative fluorescence reflects that the device works well.

<div class="image"><img src="images/R-012-color-change-of-bacteria-after-AHL-3OC6HSL-stimulation.png" /></div>
Figure 12 A picture which shows the color change of the bacteria after AHL (3OC6HSL) stimulation.

<p>(2)	E7 protein</p>

<p>We ligate e7 gene with a commercialized plasmid backbone called
pUHE21, which expresses E7 protein when there is IPTG induction. </p>

<p>W3110 is a wild type strain of E.coli that can secret
colicin. According to some research, E7 lysis protein can cause the
lysis of w3110 strain membrane so that they can release colicin to the
extracellular space to kill other E.coli strain (w3110 itself can
resist colicin because of the expression of
immunoprotein).<ref>[8][9]</ref>

<p>E.coli strain W3110 with or without E7 lysis gene integrated in
plasmid pUHE21 are cultured to OD=0.5 and then induced with or without
IPTG. We collected bacteria samples every hour from the fermented
liquid and the Optical Density are measured by microplate reader. The
results are shown below.</p>

<div class="image"><img src="images/R-013-growth-condition-after-IPTG-inducing-E7-expression.png" /></div>
Figure 13   The growth condition after IPTG inducing E7 expression which shows E7 does lyse the cells.

<div class="image"><img src="images/R-014-growth-condition-of-blank-after-IPTG-stimulation.png" /></div>
Figure 14   The growth condition of the blank after IPTG stimulation which shows IPTG stimulation does not influence OD600 a lot.

<p>From Figure 13 we get to know that the expression of E7 lysis
protein can induce the lysis of colicin-secreted E.coli. Although at
the beginning of the induction, the OD of those 3 groups are slightly
different with each other because of accidental error, after 2 hours
of induction, the differences in OD are too obvious to be considered
as the consequence of the error, and after 8 hours induction, the
difference even reaches as great as 1 fold.</p>

<p>Besides, Figure 14 can prove that IPTG is not the main cause of the
decrease growth rate of E.coli. we know that IPTG itself may do some
harm to E.coli, on the other hand, the induction with IPTG greatly
increases the expression level of genes downstream pLac ,which may
bring additional burden to E.coli as well. So in order to make sure
the difference showed in Figure 14 is mainly caused by the lysing
function of E7 lysis protein other than IPTG, w3110 without E7 lysis
gene on its plasmid (pUHE21, also referred to as backbone) is induced
with same concentration of IPTG. The result shows that IPTG can affect
the growth rate of E.coli to some degree, but its influence is not
strong enough to cause 1 fold difference. Thus we can say that E7
lysis protein is functional to cause the lysing of E.coli and decrease
its growth rate.</p>

<p>We also use the collected bacteria and the supernatant to do
western-blot,using anti-DnaK antibody. (DnaK, or Heat Shock Protein
70 is coded by a house-keeping gene and it cannot go through the cell
membrane freely.) We find DnaK in the supernatant largely increases
when E7 expresses, which means that the E7 protein do lyse the
bacteria.</p>

<div class="image"><img src="images/R-015-Western-blot-using-anti-DnaK.png" /></div>
Figure 15  Western-blot using anti-DnaK. The minus symbol means without IPTG induction. In the stimulation groups, the DnaK level increase in the supernatant shows that the E7 lyse the cell continuously.

</div>

<div class="reference">
<h2>Reference</h2>

<ul>
<li>
[1] Yurimoto H, Hirai R, Matsuno N, et al. HxlR, a member of the DUF24 protein family, is a DNA-binding protein that acts as a positive regulator of the formaldehyde-inducible hxlAB operon in Bacillus subtilis[J]. Molecular microbiology, 2005, 57(2): 511-519.
</li>
<li>
[2] Huyen N T T, Eiamphungporn W, M&auml;der U, et al. Genome-wide responses to carbonyl electrophiles in Bacillus subtilis: control of the thiol-dependent formaldehyde dehydrogenase AdhA and cysteine proteinase YraA by the MerR-family regulator YraB (AdhR)[J]. Molecular microbiology, 2009, 71(4): 876-894.
</li>
<li>
[3] Yasueda H, Kawahara Y, Sugimoto S. Bacillus subtilis yckG andyckF Encode Two Key Enzymes of the Ribulose Monophosphate Pathway Used by Methylotrophs, and yckH Is Required for Their Expression[J]. Journal of bacteriology, 1999, 181(23): 7154-7160.
</li>
<li>
[4] Sheng B, Zheng Z, Lv M, et al. Efficient Production of (R)-2-Hydroxy-4-Phenylbputyric Acid by Using a Coupled Reconstructed d-Lactate Dehydrogenase and Formate Dehydrogenase System[J]. PloS one, 2014, 9(8): e104204.
</li>
<li>
[5] Khangulov S V, Gladyshev V N, Dismukes G C, et al. Selenium-containing formate dehydrogenase H from Escherichia coli: a molybdopterin enzyme that catalyzes formate oxidation without oxygen transfer[J]. Biochemistry, 1998, 37(10): 3518-3528.
</li>
<li>
[6] Zinoni F, Birkmann A, Stadtman T C, et al. Nucleotide sequence and expression of the selenocysteine-containing polypeptide of formate dehydrogenase (formate-hydrogen-lyase-linked) from Escherichia coli[J]. Proceedings of the National Academy of Sciences, 1986, 83(13): 4650-4654.
</li>
<li>
[7] Winson M K, Camara M, Latifi A, et al. Multiple N-acyl-L-homoserine lactone signal molecules regulate production of virulence determinants and secondary metabolites in Pseudomonas aeruginosa[J]. Proceedings of the National Academy of Sciences, 1995, 92(20): 9427-9431.	
</li>
<li>
[8]Lin L J R, Liao C C, Chen Y R, et al. Induction of membrane permeability in Escherichia coli mediated by lysis protein of the ColE7 operon[J]. FEMS microbiology letters, 2009, 298(1): 85-92.
</li>
<li>
[9] Chak K F, Kuo W S, James R. Cloning and characterization of the ColE7 plasmid[J]. Journal of general microbiology, 1991, 137(1): 91-100.</p>
</li>
</ul>

</div>

</div>
</section>

<section id="modeling" class='current'>
<div>
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});
</script>
<script type="text/javascript"
  src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>

<p>If you want to look into details about the functions of our
project, please enter the following models:</p>

<h2>Formaldehyde promoter model</h2>

<div class="model-objective">
<a name="model-objective"></a>
<h3>Objective</h3>

<p>To predict the effect of the P- formaldehyde on recombinant plasmid
in Escherichia coli, we develop a mathematical model that describes
the strength of the P- formaldehyde, and in this model we simulate the
procedure of which P- formaldehyde is activated and then expresses the
downstream genes, whose expression intensity indicates the strength of
P- formaldehyde.</p>

</div>

<div class="model-description">
<a name="model-description"></a>
<h3>Model Description</h3>

<p>1.1The regulation mechanism of formaldehyde operon (HxlAB operon)
HxlAB operon encodes two key enzymes in the ribulose monophosphate
pathway that are involved in formaldehyde fixation,
3-hexulose-6-phosphate synthase and
6-phospho-3-hexuloisomerase. Expression of the hxlAB operon is induced
by the presence of formaldehyde.</p>

<img />Figure.1formaldehyde
operon (HxlAB operon)

<p>HxlR protein can enhance the operon to some extent. The hxlR gene
that encodes HxlR protein is located upstream of the hxlAB
operon. When the hxlR protein binds BRH1 operating site, the complex
can promote the binding of another hxlR protein with BRH2 operating
site. On the contrary, binding of hxlR to BRH2 operating site can not
promote the binding of BRH1 site. In other words, the binding is in a
specified order. Reactions below are involved in the regulation
process of HxlAB operon.</p>

\[ hxlR+BRH1\rightarrow hxlR\cdot BRH1(=STR1)\]
\[ STR1+BRH2+hxlR\rightarrow 2hxlR\cdot BRH1\cdot BRH2(STR2) \]
\[ STR1+STR2+TRP\rightarrow BRH1+BRH2 \]

<p>Explanation of the symbols used above: hxlR is for the protein that
has enhancing function, BRH1 and BRH2 are for two operating sites, TRP
is for the inactive operating sites.</p>

<h4>1.2 Transcription efficiency factor \(\eta\) </h4>

<p>Based on the equation above we can deduce the following equations.</p>

\[ \frac{[hxlR]\cdot [BRH1]}{[STR1]}=K_{1} \]
\[ \frac{[STR1]\cdot [BRH2]\cdot [hxlR]}{[STR2]}=K_{2} \]
\[ \frac{[TRP]\cdot [STR1]\cdot [STR2]}{[BRH1]\cdot [BRH2]}=K_{3} \]

<p>And the equations of material balance</p>

\[ [hxlR_{t}]=[hxlR]+[STR1]+[STR2]+[BRH2] \]
\[ [O_{t}]=[BRH1]+[BRH2]+[TRP] \]

<p>Explanation of symbols used above: \( [hxlR_{t}]\) is for the total
concentration of the hxlR enhancing protein, \( [O_{t}]\) is for the
total concentration of theoperating sites.</p>

<p>Given that there is constitutive instability inside the cell, the
aggregate level of operon \( [O_{t}]\) is identical to that of
heterologous gene[G]</p>

\[ [O_{t}]=[G]\]

<p>The quantity of transcription efficiency factor is defined by the
ratio of \( [O]\) and \( [O']\). According to the equation above, we
learned that</p>

\[ \eta =\frac{[O]}{[O']}=\frac{\sqrt{[B(A-G)+C]^{2}+4\cdot B\cdot C\cdot G}-[B(A-G)+C]}{2\cdot B\cdot C} \]

<p>A, B, C, and G are defined as:</p>

\[ A=[hxlR_{t}] \]
\[ B=\frac{[hxlR]^{2}}{K_{1}\cdot K_{2}\cdot K_{3}}\]
\[ C=1+\frac{[hxlR]}{K_{1}}+\frac{[hxlR]^{2}}{K_{1}\cdot K_{2}} \]
\[ G=[G] \]

<h4>1.3 mRNA concentration</h4>

<p>Supposing that the transcription and the degradation process of
mRNA inside the cell are both first order reactions and considering
the dilution effect imposed on mRNA by the growthand multiplication of
the cells, we learned that</p>

\[  \frac{d[mRNA]}{dt}=\eta \cdot k_{m}\cdot [G]-k_{-m}\cdot [mRNA]-\mu \cdot [mRNA]\]

<p>Explanation of symbols used above: \(k_{m}\) is for mRNA
transcription rate constant, \(k_{-m}\) for degradation rate constant,
\([mRNA]\) for the level of mRNA inside the cell, and for the velocity
of cells' growth.</p>

<p>Considering the transcription of mRNA in the cell is basically
quasi-stable, so</p>

\[ \frac{d[mRNA]}{dt}=0 \]

<p>mRNA level can be derived by the equation above</p>

\[ [mRNA]=\frac{\eta \cdot R_{m} \cdot G}{k_{-m}+\mu} \]

<h4>1.4The quantity of expression products in the cell</h4>

<p>Assuming that the expression of products as well as the degradation
of mRNA is a first order reaction of the quantity of mRNA inside the
cell, and considering the dilution effect imposed on the products by
the growth and multiplication of the cells, we learned that</p>

\[ \frac{d[P]}{dt}=k_{p}\cdot [mRNA]-k_{-p}\cdot [P]-\mu \cdot [P] \]

<p>Given that the expression of the products inside the cell is
basically stable, thus:</p>

\[ \frac{d[P]}{dt}=0 \]

<p>According to the equation above, we can solve for the level of cell
products</p>

\[ [P]=\frac{k_{p}\cdot [mRNA]}{k_{-p}+\mu} \]
\[ [P]=\frac{|\sqrt{(B(A-G)+C)^2+4\cdot B\cdot C\cdot G}-[B(A-G)+C]|\cdot k_{m}\cdot k_{p}}{2\cdot B\cdot(k_{-m}+\mu )\cdot(k_{-p}+\mu )\cdot\rho } \]

<h4>1.5 The concentrationof products during the fermentation process</h4>

<p>According to the equation above, we can figure out the
concentration of products in the fermentation broth, which can be
described as</p>

\[C_{p}=\frac{|\sqrt{(B(A-G)+C)^2+4\cdot B\cdot
C\cdot G}-[B(A-G)+C]|\cdot k_{m}\cdot k_{p}\cdot x}{2\cdot
B\cdot(k_{-m}+\mu )\cdot(k_{-p}+\mu )\cdot\rho }\]

<p>Explanation of symbols used above: C is for the level of the
products during fermentation process (\(\mu\)/L), x is for the level
of cells in the fermentation broth (g/L), and I is for the density of
cells (g/L)</p>

<p>From the result given above we can infer that only when the
promoter is activated, can hxlR accumulate constantly. Besides, the
probability of its binding with BRH1 and both BRH1 and BRG2 will
increase, and as a result, the metabolites in bacteria cells will
increase. </p>

</div>

<div class="model-result">
<a name="model-result"></a>
<h3>Result and Analysis</h3>

<p>In our actual experiments, we use RFP as downstream expression
product (final metabolite of cells). After stimulating the growing
bacteria with formaldehyde, we measure the fluorescence intensity of
bacteria every once in a while, the results (the greater the
fluorescence intensity, the greater the expression of RFP) are shown
in Figure 1.</p>

<img />Figure 2 The experiment of formaldehyde stimulation

<p>As can be seen from Figure 1, after the stimulation of
formaldehyde, the fluorescence intensity increases significantly,
indicating that the model predictions matches the result of
experiment, which also proves the correctness of our modeling.</p>

</div>

<h2>Reference</h2>

<ref>[1]A first course in mathematical modeling / Frank R. Giordano
... [et al.] China Machine Press, 2009.</ref>

<ref>[2] A Guide to MATLAB : For Beginners and Experienced
Users. Cambridge Cambridge University Press, 2012.</ref>

</div>
</section>

<section id="achieve" class='hidden'>
<div>

<a name="judging"></a>
<h3>iGEM Medals for Environment Teams</h3>

<h4>Requirements for a Bronze Medal:</h4>

<p>Register the team, have a great summer, and plan to have fun at the
Giant Jamboree.</p>

<p>Successfully complete and submit this iGEM 2014 Judging form.</p>

<p>Create and share a Description of the team's project using the iGEM
wiki and the team's parts using the Registry of Standard Biological
Parts.</p>

<p>Plan to present a Poster and Talk at the iGEM Jamboree. </p>

<p>The description of each project must clearly attribute work done by
the students and distinguish it from work done by others, including
hosting labs, advisors, instructors, sponsors, professional website
designers, artists, and commercial services. Please see the iGEM 2011
Imperial College Acknowledgements page for an example. Link to page on
your team's
wiki: <a href="http://2014.igem.org/Team:WHU-China/team#attribution">Attribution</a></p>

<p>Document at least one new standard BioBrick Part or Device used in
your project/central to your project and submit this part to the iGEM
Registry (submissions must adhere to the iGEM Registry
guidelines). Please note you must submit this new part to the iGEM
Parts Registry. Please see the Registry help page on adding new
parts. A new application and/or outstanding documentation
(quantitative data showing the Part's/ Device's function) of a
previously existing BioBrick part also counts. Please see the Registry
help page on how to document your contributions. To fulfill this
criterion, you will also need to submit the part with its original
part name to the Registry, following the submission guidelines.
</p>

<p>Part Number(s):</p>
<pre>
BBa_K1334002	[Received, Accepted]
BBa_K1334020	[Received, Accepted]
BBa_K1334000	[Received, Accepted]
BBa_K1334001	[Received, Accepted]
BBa_K1334003	[Received, Accepted]
BBa_K1334004	[Received, Accepted]
BBa_K1334005	[Received, Accepted]
BBa_K1334006	[Received, Accepted]
BBa_K1334007	[Received, Accepted]
BBa_K1334008	[Received, Accepted]
BBa_K1334015	[Received, Accepted]
BBa_K1334016	[Received, Accepted]
BBa_K1334017	[Received, Accepted]
BBa_K1334018	[Received, Accepted]
BBa_K1334019	[Received, Accepted]
BBa_K1334020	[Received, Accepted]
BBa_K1334021	[Received, Accepted]
BBa_K1334022	[Received, Accepted]
BBa_K1334024	[Received, Accepted]
</pre>

<h4>Additional Requirements for a Silver Medal:</h4>

<p>Experimentally validate that at least one new BioBrick Part or Device of your own design and construction works as expected.</p>

<p>Part Number(s):</p>
<pre>
<a href="http://parts.igem.org/Part:BBa_K1334002">BBa_K1334002	[Received, Accepted]</a>
<a href="http://parts.igem.org/Part:BBa_K1334020">BBa_K1334020	[Received, Accepted]</a>
</pre>

<p>Document the characterization of this part in the Main Page section
of that Part's/Device's Registry entry.  (the same linkage as
above)</p>

<p>Submit this new part to the iGEM Parts Registry (submissions must
adhere to the iGEM Registry guidelines)</p>

<p>Part Number(s):</p>
<pre>
<a href="http://parts.igem.org/dna_transfer/one_batch.cgi?batch_id=2455">BBa_K1334002	[Received, Accepted]</a>
<a href="http://parts.igem.org/dna_transfer/one_batch.cgi?batch_id=2455">BBa_K1334020	[Received, Accepted]</a>
</pre>

<p>iGEM projects involve important questions beyond the bench, for
example relating to (but not limited to) ethics, sustainability,
social justice, safety, security, or intellectual property
rights. Articulate at least one question encountered by your team, and
describe how your team considered the(se) question(s) within your
project. Include attributions to all experts and stakeholders
consulted. Link to page on your team's wiki: <a style="background: red" href="http://2014.igem.org/Team:WHU-China/practice">Research: Formaldehyde in cars</a></p>

<h4>Additional Requirements for a Gold Medal: (one OR more)</h4>

<p>Improve the function OR characterization of an existing BioBrick Part or Device (created by another team or your own institution in a previous year), enter this information in the Registry. Please see the Registry help page on how to document a contribution to an existing part.</p>

<p>Part Number(s):</p>
<pre>We contribute to the part <a href="http://parts.igem.org/Part:BBa_K117000:Experience">BBa_K117000</a>.
</pre>

<p>Help any registered iGEM team from another school or institution by, for example, characterizing a part, debugging a construct, or modeling or simulating their system.
Link to page on your team's wiki: <a style="background: red" href="">Helpling Other Teams</a></p>

<p>iGEM projects involve important questions beyond the bench, for
example relating to (but not limited to) ethics, sustainability,
social justice, safety, security, or intellectual property
rights. Describe an approach that your team used to address at least
one of these questions. Evaluate your approach, including whether it
allowed you to answer your question(s), how it influenced the team's
scientific project, and how it might be adapted for others to use
(within and beyond iGEM). We encourage thoughtful and creative
approaches, and those that draw on past Policy & Practice (formerly
Human Practices) activities. Link to page on your team's wiki:
<a style="background: red" href="">Research: Formaldehyde in cars</a></p>

</div>
</section>

<section id="practice" class='hidden'>
<div>

<div class="research">
<a name="research"></a>
<h2>Research: Formaldehyde in cars</h2>

<h3>Concentrating On the Safety, Leading Our Design</h3>

<p>Before we started our project "formaldehyde terminator", we aimed
at detecting the existence of formaldehyde in the houses, but as time
went by, we realized that high formaldehyde level is a more serious
problem in cars. "Although we can tell whether the level of
formaldehyde is high or not, we have almost no methods to eliminate
formaldehyde in cars quickly." So we improved our design by adding
Street Cleaner System, so we could not only detect formaldehyde but
also remove it continuously. </p>

<h3>Introduction</h3>

<p>As we know, formaldehyde exists not only in factories and some
extreme environment but also in our daily life, but at the same time
we often ignore the formaldehyde in some basic area around us, we aim
nat finding what we used to ignore, and finally we have done the
research of detecting formaldehyde in cars. In this program, we use
formaldehyde kits to help both teachers and our classmates to detect
formaldehyde level in their cars. 34 cars have been tested and finally
almost one third of these cars’ air-formaldehyde condition is
excessive. Besides these, we have also done some other analysis about
different factors’ influence on formaldehyde level.</p>

<p>You can download the detail information
from <a href="">here</a>.</p>

<div class="line-wrapper">

<div class="image"><img src="images/H-photo-01.jpg" /></div>

<div class="image"><img src="images/H-photo-02.jpg" /></div>

</div>
<div class="clearfix"></div>

<h3>Analysis</h3>
<p>After we have collected series of data, we did a lot of
analysis. For the reason that our data are influenced by different
factors, we set groups of two-variables analysis (other factors are
ignored when analyzing one group). Here are our data:</p>

<div class="image"><img src="images/Table 1.1.jpg" />
<p>Table 1.1 The rate of excessive cars and compliance cars</p>
</div>

<div class="image"><img src="images/H-figure-1-1.jpg" />
<p>Figure1.1 The rate of excessive cars and compliance cars</p>
<p>The cars which have excessive level of formaldehyde are about one third to all cars tested.</p>
</div>
 
<div class="image"><img src="images/H-figure-1-2.jpg" />
<p>Figure1.2 Time and cars Red line stands for Cordon - It is the content of formaldehyde higher than 0.1mg/m<sup>3</sup> that is excessive.</p>
</div>

<p>Using years is related to level of formaldehyde in cars closely,
through the graph we can find that there exists a clear boundary
between the excessive group and the normal group, new cars are of high
possibility to have excessive level of formaldehyde while old cars may
remain normal. (For the reason that quantity of our data is limited
and unknown factors may also influence the research, we can only get
the rough conclusion)</p>

<div class="image"><img src="images/Table 1.2.jpg" />
<p>Table 1.2 Countries and cars</p>
</div>

<div class="image"><img src="images/H-figure-1-3.jpg" />
<p>Figure1.3 Rate of cars produced from different countries</p>
</div>

<p>Besides the quantity, we also did the analysis of the relationship
between countries and average formaldehyde level of cars.</p>
 
<div class="image"><img src="images/H-figure-1-4.jpg" />
<p>Figure1.4 Average level of formaldehyde in different groups.</p>
<p>From this graph we can know clearly that cars from different countries have diversity in the level of formaldehyde, so we may think it over before deciding which one to buy.</p>
</div>

<h3>Conclusion</h3>

<p>From this program we have finished, we can find that the basic
vehicle - cars may have higher concentration of formaldehyde than we
thought before (For the reason that nearly 30 percent of cars may have
excessive level of formaldehyde). It alerts us that we should focus
our attention on what we may ignore before, even the basic one around
us. Through this program, teachers and our classmates have got access
to our iGEM project, almost all of them participated in it happily and
finally found what was ignored before. This is not only good for
communication between teachers and students, but also can help more
people to realize environmental problems in our daily life.  We feel
sorry for that kits can only detect but not remove formaldehyde at the
same time, if our iGEM project could come true, we can solve this
problem easily. More than one teacher have shown their interest in our
project of using E. coli to detect and remove formaldehyde at the same
time, we will work harder and try all we can to make our dream come
true!</p>

</div>

<div class="helping">
<a name="helping"></a>
<h2>Helping other teams</h2>

<h3>Giving iGEM14_NUDT_CHINA A Biobrick We Constructed</h3>

<p>We give iGEM14_NUDT_CHINA a new biobrick we constructed this
year. The part number
is <a href="http://parts.igem.org/Part:BBa_K1334030">BBa_K1334030</a>. They
used this part in their project and test its function. Thanks for
their acknowledgement in
theri <a href="http://2014.igem.org/Team:NUDT_CHINA/Attributions">Experiment
Support</a>.
</p>

<h3>Helping iGEM14_HUST-INNOVATORS Constructing Their Biobrick</h3>

<p>In September in 2014, after we finished our project probably and
went to the last step, we helped iGEM14_HUST-INNOVATORS construct a
biobrick BBa_K1551000.</p>

<p>The
biobrick: <a href="http://parts.igem.org/Part:BBa_K1551000">BBa_K1551000</a>.</p>

<p>The reference page
of <a href="http://2014.igem.org/Team:HUST-Innovators">iGEM14_HUST-INNOVATORS</a>
(Please click <em>Team-intro.-Special Thanks</em>)</p>

<p>iGEM14_HUST-INNOVATORS is a team which consists of intelligent guys
whose major are not biology-relevant. They made a piece of fascinating
software and found a biological company to synthesize a part in pUC57
backbone to generate delta-5 fatty acid Elongace while they had
difficulties in transforming the part into pSB1C3. We changed the
backbone for them and helped them by adding the part to the
registry.</p>
 
<div class="image"><img src="images/H-figure-2-1.jpg" />
<p>Figure2.1 Detail information of plasmid pUC57</p>
</div>

<p>After days of hard working, we finished this project successfully,
here shows the final result of electrophoresis.</p>

<div class="image"><img src="images/H-figure-2-2.png" />
<p>Figure 2.2 We digest Bba_K1551000 with restriction enzymes EcoR I and
PstI . The Elongace coding sequence (with prfix and suffix)is 2064 bp
while the pSB1C3 is 2070 bp so that there is only one bar near the
2000 bp region, which shows that we change the plasmid backbone
successfully.</p>
</div>

<h3>Participating in other teams' human practice</h3>

<p>In June, we got an e-mail from the team Virginia, from this e-mail
we knew that they were preparing for a global survey about synthetic
biology, we participated in their survey actively and finally helped
them to collect nearly 50 surveys, they provided us with a nice badge
and thanked us for helping.</p>

<div class="image">
<p>Email from Virginia:</p>
<img class="framed" src="H-photo-03.jpg" />
</div>

<div class="virginiabadge">
<style>
#virginiabadge { margin: auto; }
</style>

    <div id="virginiabadge">
        <a href=
        "http://2014.igem.org/Team:Virginia/HumanPractices"><div class="image"><img height="180"
        src="http://2014.igem.org/wiki/images/b/b0/Virginia_Transparent.png"
        width="250"></a>
    </div>

    <style>
      #virginiabadge{
      background: url("http://2014.igem.org/wiki/images/6/6a/Virginia_Badge2.png");
      background-size:250px;
      width: 250px;
      height: 185px;
      background-repeat:no-repeat;
    }
 
    #virginiabadge:hover{
     background: url("http://2014.igem.org/wiki/images/2/2b/File-Virginia-Images-Badge1.png");
     background-repeat:no-repeat;
     width: 250px;
     height: 185px;
     background-size:250px;
    }
    </style>

</div>

<p>Besides, we also helped XMU-China to finish their Newsletters, we
were very happy to see their Newsletter every week.</p>

<div class="image">
<p>Email from XMU-China:</p>
<img class="framed" src="H-photo-04.jpg" />
</div>


<h3>Helping other teams after CCiC:</h3>

<p>Besides the information mentioned above, after the CCiC(Central
China iGEMer's Consortium), we amplified our communication with other
teams such as HZAU-China, NJAU-China, NUDT-China and so on, we also
provided them with help, here is the brief introduction:</p>

<p>Before the CCiC, we convened a preparation meeting in WHU-China,
and helped HZAU-China to formed CCiC. We also exchanged some parts
with HZAU-China, such as dCas9 device(BBa_K1081000)(we constructed
this part last year), and luxI generator： BBa_K1334030(we constructed
the part this year)</p>

<p><a href="http://2014.igem.org/Team:HZAU-China/acknowledgment">Special
thanks from HZAU-China</a></p>

<p>After communicating with NJAU-China, we knew that their program is
related to dCas9 device, which we had constructed the last year, so we
also provided them with BBa_K1081000.</p>

<p><a href="http://2014.igem.org/Team:NJAU_China/Collaboration">Thanks from NJAU-China</a></p>

<p>After communicating with NUDT-China, we knew that both of our
projects would use N-(3-oxododecanoyl) homoserine lactone (AHL for Las
system), at that time we had gotten AHL, so we sent AHL to them,
besides, we provided BBa_K1334030 to them. Similar situation happened
to BIT, which team had also received AHL from us.</p>

<p><a href="http://2014.igem.org/Team:NUDT_CHINA/Attributions">Address from NUDT_CHINA</a></p>

<p><a href="http://2014.igem.org/Team:BIT/human_practice.html">Address from BIT</a></p>

</div>

</div>

<div class="popularization">
<a name="popularization"></a>
<h2>Popularization </h2>

<p>During our process in finishing the project, we had also tried to
introduce iGEM to more people around. To achieve our goal, we had
hosted series of wonderful activities.</p>

<p>In May, at the very beginning of our project, we hosted Scrawl
Competition in Microorganism, in this competition, every participant
would get a culture plate and E. coli which contained GFP, they could
use inoculating loop to draw what they like on the culture plate.</p>
   
<div class="image"><img src="images/H-photo-05.jpg" /></div>
<div class="image"><img src="images/H-photo-06.jpg" /></div>
<div class="image"><img src="images/H-photo-07.jpg" /></div>



<p>In September, we utilized the chance of welcoming the fresh to our
university to advertise iGEM, in order to bring them knowledge about
iGEM as well as happiness, we hosted series of activities. When the
fresh come to university to register, they will get a bag which
contains books about the university, card they will use in daily life
and other guide that can help them to adapt the university. We added
our poster and an exquisite badge into each bag, they were very happy
when getting gifts. In addition, we also hosted a meeting to welcome
them and advertise iGEM, our header Sun Haoyuan gave them a wonderful
speech and told them a lot about his experience in university and
iGEM.</p>
       
<div class="image"><img src="images/H-photo-08.jpg" /></div>
<div class="image"><img src="images/H-photo-09.jpg" /></div>
<div class="image"><img src="images/H-photo-10.jpg" /></div>

<p>Further than advertising to students around us, we have taken other
ways to expand our influence. One way is through the internet, we have
set up our own Xinlang Weibo, public pages on Renren and QQ groups, we
keep telling the public about iGEM and the process of our
experiments. Till now on ,we have a lot of fans, and they can learn
numerous knowledge from us. Another way is participating in
competition hosted by our school, in the early October, we took our
program to Creation Competition in our school, and we showed our idea
to many teachers in different majors and finally got a great
score.</p>

</div>

<div class="communication">
<a name="communication"></a>
<h2>Communication</h2>

<p>We have communicated with a lot of universities and
organizations. First of all we will introduce you CCiC here. CCiC is
Central China iGEMer's Consortium. In the early April, HZAU-China,
HUST-China and WHU-China ,WHU-Pharm worked together to prepare for
this great meeting, and CCiC finally began at 23rd August successfully
in Huazhong Agriculture University. Almost 20 universities from all
over the China came to Wuhan to attend the meeting. We made friends
with students from these universities and at the same time we
communicated and exchanged our great ideas with each other.</p>
 
<div class="image"><img src="images/H-photo-11.jpg" /></div>
<div class="image"><img src="images/H-photo-12.jpg" /></div>
<div class="image"><img src="images/H-photo-13.jpg" /></div>

<p>Besides CCiC, we also communicated with a lot of teams. Here is a
list that notes the wonderful experience:</p>

<p>Our leader went to Jilin University and had a nice communication
with them, besides exchanged our ideas, he invited Jilin_China to join
in CCiC and have a visit to Wuhan University.</p>

<div class="image"><img src="images/H-photo-14.jpg" /></div>

<p>Our advisor Jia Hangxing was invited by friends from SCU to visit
SiChuan University and they had a wonderful talk in May.</p>

<div class="image"><img src="images/H-photo-15.jpg" /></div>

<p>Friends from OUC-China, SCUT and SCAU-China visited Wuhan
University after CCiC. Students from WHU-China and WHU-Pharm welcomed
them and guided them a wonderful trip.</p>

<div class="image"><img src="images/H-photo-16.jpg" /></div>

</div>

</div>
</section>

<section id="team" class='hidden'>
<div>

<div class="members">
<a name="members"></a>
<h2>Members</h2>

<div class="image"><img src="images/T-photo-1.jpg" /></div>

<p>Hello everyone, Here is the team of WHU-China!  Team WHU-China 2014
was born at the foot of the mount LuoJia, November 2013. The team is
mainly formed by sophomore and junior of the college of life sciences,
Wuhan University. Our team has a great tradition and this year is the
third time to join in the competition. In our warm small group, both
sophomore and junior work together. We hope that we can learn more
knowledge of synthetic biology and get access to more experimental
skills by ourselves during the competition. What is more important is
that we can make a contribution to the society. Let us join hands to
swim in the vast ocean of synthetic biology and find treasures one by
one!</p>

<div class="image"><img src="images/T-photo-2.jpg" /></div>

<p>Haoyuan Sun, as the captain of the team, is devoted himself into
the completion. From the establishment of the project, to undertake
the whole experiment guideline, he is just like a glaring star in the
night to guide us where to go ahead. In the iGEM team, he not only
uses his knowledge to solve the problems, but also encourages us by
his humor to get through the difficulties. Besides the iGEM, he is
skilled in playing ball games and telling jokes.</p>

<div class="image"><img src="images/T-photo-3.jpg" /></div>
 
<p>Hi, I am Zheng Sun, In the team of iGEM, everyone can find his/her
own place, and iGEM has different meanings to different people. As for
me, iGEM provides me with a great chance to explore the field of
synthetic biology and the way to go into the society to touch
different people and handle series of things I have never met
before. Thanks iGEM for giving me such a chance and I believe this
wonderful experience can leave me the beautiful memory and treasure in
my life.</p>

<div class="image"><img src="images/T-photo-4.jpg" /></div>
 
<p>Here is Yun Xue, this is my first year in iGEM. I joined iGEM
because I'm fond of synthetic biology and I'm skilled in the
mathematical modeling, genetics and molecular biology. And I would
like to pursue these interests in future and do research about
mathematical modeling of biological systems. I enjoy sport activities
such as badminton, tennis and basketball. I am lookng forward to
solving problems and getting good achievements.</p>

<div class="image"><img src="images/T-photo-5.jpg" /></div>
 
<p>Qiushuang Wu: the smart girl provided us with the original idea for
our project this year in the brainstorming. As the housekeeper in our
lab, she kept the lab organized when doing experiments. And she
herself did a lot of experiments, involving all parts in our
project. As some people's comment, she was always in the lab or on
the way to the lab. She usually says if love, please love deeply. She
loves biology very much, and will devote to her true love all her
life.</p>

<div class="image"><img src="images/T-photo-6.jpg" /></div>
 
<p>Long Yu, coming from Hubei, China, majors in biology in College of
Life Sciences, Wuhan University. In his second year, he entered into
the lab of Prof. Chen, whose field is microbiology. During those
times, he was mainly devoted to reveal a new kind of mechanism by
which genetic materials were transferred between Bacilus subtilis
other than three traditional horizontal gene transfer (HGT) ways. In
his third year, he participated in this iGEM team, and devoted much
time into it, so granted, he learned a lot. But it is a pity that he
could not keep it up. Now in his fourth year in college, he was
prepared to go to Monash University in Australia, conducting
researches about Tfh cells.</p>

<div class="image"><img src="images/T-photo-7.jpg" /></div>
 
<p>My name is Han Dong. I have been studying in Wuhan University for
more than 2 years, majoring in biology. I was born in a coastal city
named Qingdao, where I grew up with the sea, thus I like seafood,
beach and sea breeze. Sea makes me open-minded, willing and have broad
interests like swimming. I also enjoy reading and drawing, so I am the
main graphic designer in our team. I believe that painting will help
people to learn the world better by watching it carefully, and it is
the same with biology. That is why I join the team , I want to show
our idea with my pen.</p>

<div class="image"><img src="images/T-photo-8.jpg" /></div>
 
<p>I am Teng Li, a girl who loves reading and wants to make everyday
in my life meaningful. I regard being an iGEMer a happy thing that
deserves my devotion. All the iGEMers in our team are my friends and I
do enjoy performing experiments and exchanging ideas with them. It is
in iGEM that I learned how to cooperate, how to think and the basic
skills for doing research of synthetic biology.</p>

<div class="image"><img src="images/T-photo-9.jpg" /></div>

<p>Hello! I am Zhengyu Luo, a senior student major in math and I am
the computer engineer of the team in charge of the construction of the
webpage. I like art and design and I am full of energy and prepared
for any kind of challenges. Synthetic biology is a very hot subject
these days so I joined WHU-China to learn something about it. I
believe it must be a very interesting experience.</p>
 
<div class="image"><img src="images/T-photo-10.jpg" /></div>

<p>Hey guys! I'm Li Yuwenbin from Wuhan University, a team member of
Whu-China. My job in our team mainly focuses on proving the function
of E7 lysis protein as well as integrating it into our
formaldehyde-terminator system, so that those functional enzymes can
be secreted out into extracellular space to degrade formaldehyde more
efficiently.  Besides biological research, I have a strong passion for
various kinds of sports, like cycling, badminton and even x-game. I
also have a special taste for music, poems and animation. iGEM
jamboree has a lot of fun and it's so great to meet new friends and
exchange ideas with each other in the competition. See you guys at
MIT^ ^!</p>

<div class="image"><img src="images/T-photo-11.jpg" /></div>
 
<p>Hi there, this is Tianxing Zhai, a student of College of Life
Sciences at Wuhan University. As a team member of WHU-China, I was
responsible for the functional verification of formaldehyde
dehydrogenase (spectrophotometry), some molecular cloning, and other
non-experimental activities (e.g. ordering uniforms and badges). I am
interested in making science more interesting and more widespread. If
you and I are like-minded, please contact me. My Email is
txzhai@whu.edu.cn.</p>

<div class="image"><img src="images/T-photo-12.jpg" /></div>
 
<p>Hi, I'm Bei Liu. I was in the lab of proteomics in Wuhan
University. I did some experiments of biochemistry in the early
stage. I also helped Haoyuan in organizing the team and putting up
some ideas. Besides synthetic biology, I'm also interested in
proteomics and neurogenesis. Now I'm doing an internship in UT
Southwestern medical center in Dallas. It is really cool to
participate in the giant jamboree in Boston, because Dallas is just
like a large valley, while Boston is the true Big City. I hope I could
spend some time visiting the city in the spare time during the
jamboree.</p>

<div class="image"><img src="images/T-photo-13.jpg" /></div>
 
<p>I am Yao Chen, I still remember the first time I knew something
nabout iGEM when I was a freshman, my classmate joined this team
whereas I could only listen to his proud talking. Until last November
I participated in the iGEM team for my dream. We experienced the hard
Saturday night brainstorming and staying up late for presentation of
our project and finally our sub-team project became the only project
in the team. First I even could not understand what we would do but
thanks for the team members, I made it clear ultimately.  To
retrospect the last 10 months we went through the hardness and success
which enrich my experience, broaden my horizon and give me
friends. Now I am placid to write the story between iGEM and me and
appreciate all the people helped us.</p>

<div class="image"><img src="images/T-photo-14.jpg" /></div>
 
<p>I am Shuohui Liu. During the four years in university, one could
choose to sit around and achieve nothing. Or one could treasure the
time to do something meaningful. The most meaningful thing for me is
to join the iGEM. In the iGEM, one could do researches that are really
special and interesting. One could also make friends and work in a
team efficiently and comfortably. What's more, one could enjoy the
best time in the life. In other words, the one in the iGEM is the most
fortunate one in the word. Absolutely, I'm a guy like that.</p>

<div class="image"><img src="images/T-photo-15.jpg" /></div>

<p>Zhengwei Yuan| Senior in Biological Sciences Zhengwei is
responsible for the separation of FDH (Formate dehydrogenase) gene
from Candida boidinii which was bought from China Center of Industrial
Culture Collection (CICC) and he work so hard with our team leader
that they stay up late for three consecutive days to do the
experiment. The main driving force of their diligence is the love for
the iGEM and members in our team!  </p>

<div class="image"><img src="images/T-photo-16.jpg" /></div>
 
<p>I joined iGEM-team at my second year of colleague. I chose
life-science and wished to learn how to design a new life by
myself. iGEM gives me the chance to know more and do more. Ideas will
never appear suddenly - after the brain storm and layers of screening
we got the amazing idea finally and had a colorful life in the lab for
several mouths. I joined the human practice group at the last few
weeks and did some help. Thanks to my teammates and teachers, we
finished it quite well. I love dancing and reading novels. My favorite
novel is Harry Potter and I don't think it is childish. Life is
colorful, both in the lab and outside.</p>

<div class="image"><img src="images/T-photo-17.jpg" /></div>
 
<p>My name is Lei He coming from Wuhan University school of
pharmaceutical science. Because of my enthusiasm and teachers' help, I
have got a rich experience of experiment. But being an iGEMer of a
great team is the most exciting part.</p>

<div class="image"><img src="images/T-photo-18.jpg" /></div>
 
<pre>NAME			 Shuang Liu
BIRTHDAY		 1994/01/04
CITIZENSHIP              the People's Republic of China 
NATIVE PLACE             Shandong Province
HEIGHT			 5 feet 10 inches
WEIGHT			 158 lb
BLOOD TYPE               A
ALLERGY		         NONE
RELIGION		 ATHEISM</pre>

<div class="image"><img src="images/T-photo-19.jpg" /></div>

<p>Hi I'm Chen Xi or you can call me Fred. As a senior year student,
I'm grateful to be a member of this excellent miracle-making team. My
work in this project involves source gathering, plasmids design. I am
so glad that our idea and the outcome can now have a chance to be
presented to the world and let people know how the bioengineering can
create a formaldehyde toxicity free world.</p>

<div class="image"><img src="images/T-photo-20.jpg" /></div>
 
<p>My name is Ji Yang. I am a senior student in College of life
science, Wuhan University. It's a great honor to be a part of the
2014-WHU-China team. In the team, my responsibility is to assist Luo
Zhengyu to construct our web page and collect some necessary text
material. We would use some fancy CSS and JavaScript techniques to
make our page more attractive and more creative. I wish our work could
present you a clear and insight web page.</p>

<div class="image"><img src="images/T-photo-21.jpg" /></div>

<p>Qi Wang| Senior in Biology science Qi is really a shy and
introverted girl. She does not like to talk with other members very
much, but she is willing to be a good listener. A good team needs not
only someone who is eloquent enough to keep us together, but someone
who dedicated himself silently like Qi. And she is very serious with
her work. She worked in the lab for our project in the summer
vacation.</p>

<div class="image"><img src="images/T-photo-22.jpg" /></div>
 
<p>Hi! I am Man Li from WHU-China. I met iGEM one year ago and I am
totally into it. In my mind, iGEM is not a competition merely but a
club. In this club, students who have the same interests in
genetically engineering work together. We set our goals, and achieve
some ridiculous ideas. As for myself, I am an outgoing girl. For me,
research in biology is one part of my life. In my mind, the quality of
life is equally important to the achievement I make in biology. I play
violin in my spare times, and enjoy the peaceful afternoons. In
WHU-China, I am the one who mobilize the atmospheres in our laboratory
and cheer others up.</p>

<div class="image"><img src="images/T-photo-23.jpg" /></div>
 
<p>I'm Rui Bai, a senior student majored in life science, I'm so lucky
to be a part of our awesome WHU-iGEM. Before entering the team, I have
participated in a molecular biology lab for a year. Because of
curiosity, I joined in our team to broaden knowledge of synthetic
biology and share ideas with coworkers, which we called
brainstorming. Now I think it has been one of the most correct choices
which I have made in my college. What's more, I am good at organizing
activities for heading up the Student Union. And I also have an
appetite for drawing and basketball.</p>

<div class="image"><img src="images/T-photo-24.jpg" /></div>
 
<p>Xi Tu| Senior in Biological Sciences Xi joined the team, wondering
how synthetic biology works. He takes part in discuss the idea and
good at primer design.</p>


<h3>Instructors</h3>

<div class="image"><img src="images/T-photo-25.jpg" /></div>
 
<p>Zhaoning Wang</p>
 
<div class="image"><img src="images/T-photo-26.jpg" /></div>

<p>Tong Liang</p>

<div class="image"><img src="images/T-photo-27.jpg" /></div>
 
<p>Hangxing Jia</p>

<h3>Advisors</h3>

<div class="image"><img src="images/T-photo-28.jpg" /></div>
   
<p>Prof. Zhixiong Xie</p>

<div class="image"><img src="images/T-photo-29.jpg" /></div>
 
<p>Prof. Yu Chen</p>

</div>

<div class="attribution">
<a name="attribution"></a>
<h2>Attributions</h2>

<ol>
<li>
Prof. Lin Guo from the College of Life Sciences in Wuhan University provided us with lab and equipment to finish parts of our project.
</li>
<li>
Prof. Fang Peng from China Center for Type Culture Collection helped us with the culture medium's making and giving information of culturing and separating bacteria.
</li>
<li>
Prof. Yin Lei from the College of Life Sciences in Wuhan University helped us to finish the project of detecting formaldehyde in cars, he provided us with formaldehyde electronics test device. 
</li>
<li>
Friends from Huazhong Agriculture University provided us with the part BBa_J13002, and they also inspired us and entertained us very well during the CCiC.
</li>
<li>
Mr. Jiawei Huang, Yan Zeng helped us during the whole process of experiment. They provided us with a lot of new ideas and new methods.
</li>
<li>
Mr. Yang Liu helped us to find our way in finishing school internal procedures and getting visa.
</li>
<li>
Teachers from the College of Life Sciences in Wuhan University helped us to finish our program of detecting formaldehyde in cars, they participated in it actively and introduced it to their friends. Also thanks our classmates to join in this program.
</li>
<li>
Yunyun An, a very clever girl help us with our daily experiments as a technician for several days when we are busy. 
</li>
<li>
16 teams have participated in this meetup CCiC. Without them, the first Central China iGEMers' Meetup couldn't be such a happy, exciting and wonderful meetup. These 16 teams are: OUC_China, SCUT, CAU_China, WHU_China, NJU-QIBEBT, SICAU, Jilin-China, AHUT_China, Peking, LZU-China, SCAU_China, BNU-China, WHU-Pharm, HUST-China, BIT, NUDT_China, SICAU.
</li>
</ol>

</div>

</div>
</section>

<section id="safety" class='hidden'>
<div>

<div class="overview">
<p>Overview：Welcome to our Safety Page. We will first answer the
safety questions asked by iGEM headquarters briefly, and then discuss
safety issues associated with our project in detail..</p>
</div>

<div class="brief">
<a name="brief">
<h2>Brief answers to safety questions</h2>

<p>1. Have your team members received any safety training yet?  Please
briefly describe the topics that you learned about (or will learn
about) in your safety training.</p>

<p>Yes, we have already received safety training. Our team members
have been trained after Guidance of Student Experiments formulated by
TCEB (Teaching Centre of Experimental Biology, Wuhan University),
which includes how to correctly and safely use chemicals and
equipments, how to manipulate organisms including bacteria.</p>

<p>2. Who is responsible for biological safety at your institution?
(You might have an Institutional Biosafety Committee, an Office of
Environmental Health and Safety, a single Biosafety Officer, or some
other arrangement.) Have you discussed your project with them?
Describe any concerns they raised, and any changes you made in your
project based on your discussion.</p>

<p>Our lab is supervised by Teaching Centre of Experimental Biology,
Wuhan University (TCEB, whose leader is our instructor Dr. Zhixiong
Xie). An expert group of this center formulates safety guidelines and
each experiment we have done have passed its review. In our former
project, we planned to get a part from Pseudomonas putida while they
recommended us to use the genome DNA instead of the living bacteria in
the consideration of safety.</p>

<p>3. What are the biosafety guidelines of your institution? In your
country, what are the regulations that govern biosafety in research
laboratories?  These two links give the information about the our
institution guidelines and government regulations.</p>

<p>http://www.bio.whu.edu.cn/News_read.asp?cid=123&amp;id=2649</p>
<p>http://www.moh.gov.cn/mohzcfgs/pfg/200804/20386.shtml</p>

<p>4. Please describe risks of working with the biological materials
(cells, organisms, DNA, etc.) that you are using in your project. If
you are taking any safety precautions (even basic ones, like rubber
gloves), that is because your work has some risks, however
small. Therefore, please discuss possible risks and what you have done
(or might do) to minimize them, instead of simply saying that there
are no risks at all.</p>

<p>All the organisms we have dealt with and planned to
use(Escherichia. coli DH5a and Bacillus subtilis 168) are not of high
individual or community risk while there is a low probability that the
organisms will cause some diseases such as diarrhea which can be
easily treated.

The organisms we have dealt with and planned to use cannot keep alive
for a long time if they can escape from our lab. But theoretically,
they maybe survive and then cause some disease and the plasmids may
also be transformed into the other organisms in the environment.

We use some chemicals including agarose, TAE buffer, acrylamide,
ethanol, restrictive enzymes, GelRed, ampicillin, kanamycin,
chloramphenicol, tryptone, yeast extract, NaCl. Most of these are not
of high risk while some antibiotics may harm by incorrectly use.

The biological materials we use are all carefully selected for our
project and of very low risk. No one could use them maliciously to
hurt individuals and the public.</p>

<p>5. What measures are you taking to reduce these risks? (For example:
Safe lab practices, choices of which organisms to use.)</p>

<ol>
<li>Everyone in the lab has been trained for more than two years by TCEB, and we believe we can handle the basic parts of the project.</li>
<li>When we design our experiment, we have searched a great amount of paper and asked some professors about what kind of materials we can use, we can make a certain that they are available and safe.</li>
<li>Before getting organism materials and medicine, we need to acquire agreement from our administrator in the lab, and when we use them, we take some protective strategies, for example, we wear rubber gloves and experimental coat and we manipulate bacteria in ultra clean cabinet and strict aseptic techniques are followed.</li>
<li>Our lab has a complete institution for that the lab is basic-biosafety level 1 lab. All microorganism containing vessels are sterilized before and after use following appropriate protocols. We make sure that power supply in the lab are cut off before we leave the room, and we store the medicine and biological samples in order and in safe place strictly.</li>
<li>No evidence, both theoretically and experimentally, shows that our work may improve the pathogenicity of the bacteria or may cause any damage to exterior environment, even taking into consideration the risks of horizontal gene transfer.</li>
</ol>

</div>

<div class="issues">
<a name="issues">
<h2>Safety issues evaluation</h2>

<p>As stated above, all the organisms and DNA hosts are not of high
individual or community risk. Until now, E. coli DH5&alpha; is the
very one organism we have used in our experiments. Our current lab of
basic-biosafety level 1 is safe enough to deal with E.coli. To
manipulate the strain, an ultra clean cabinet is used and strict
aseptic techniques are followed. Every member of us has been strictly
trained on fundamental microbiology techniques and biosafety. All
microorganism containing vessels are sterilized before and after using
following appropriate protocols. Also, materials containging any
microorganisms will be sterilized before discarded. In this way, we
guarantee that no public or environmental harm will be caused by the
experimental organisms, and that no one in our team will get hurt
during the experimental process.</p>

<p>The genetic modifications we bring in will alter the expression of
bacterial genes. However, these genes are merely designed to test the
functions of elements we have constructed in order to achieve goals
about gene activations. No evidence, both theoretically and
experimentally, shows that these changes may improve the pathogenicity
of the bacteria or may cause any damage to exterior environment, even
taking into consideration the risks of horizontal gene transfer.
Further details about gene safety can be found in relevant parts on
our wikipage.  We have also examined the potential nocuities of
diverse chemicals and equipments as listed below: </p>

<ol>
<li><em>Basic molecular experiments:</em>
NaOH, HCl, SDS, acrylamide, TEMED, ethanol, IPTG, Restrictive enzymes from NEB, GelRed, Agarose, TAE buffer, 10XLoading buffer, BSA, 1kb DL2000 plus DNA marker, Gibson assembly reagents, CaCI2</li>
<li><em>Bacteria culture:</em>
ampicillin, kanamycin, chloramphenicol,Tryptone, Yeast extract, NaCl</li>
<li><em>Equipments:</em>
UV lamp, centrifugal machine, heating equipments (alcohol burner, PCR amplifier, water bath, dry bath), electrophoresis apparatus, -80℃ refrigerator, ultrasonic cell crusher, constant temperature incubator, shakers, microwaveoven, ice machine.</li>
</ol>

<p>All of these are common reagents and apparatus in any molecular
laboratory. Potential risks may come from inflammability,
explosibility, irritation, corrosivity, toxicity, carcinogenicity or
physical injury. Moerover, antibiotics of merely trace amount are used
in our lab and more importantly, only after special treatment will
they be discarded in the end. Nevertheless, equipped with chemical
hood and emergency shower, we can, without doubt, protect ourselves
from any kind of injury. Thus, none of them raise special safety
issues. </p>

<p>Our project has past a review of an expert committee, of which the
members are professors or associate professors of microbiology,
genetics and bioengineering. Safety issues are considered seriously
before they finnaly approved of our project. Our lab is supervised by
Teaching Centre of Experimental Biology, Wuhan University (TCEB, whose
leader is our instructor Dr. Zhixiong Xie). An expert group of this
centre formulates safety guidelines and guarantees their
implementations in all teaching labs. All experiments we have done
have also past its review. Our safety management system includes the
followings: Management of Chemicals, Equipments and
Experimenters. Chemicals and equipments are purchased instead by TCEB
and are also under tightly management. Equipments are checked and
maintained regularly and an entrance guard system is brought in to
ensure that only formal members can enter the lab.</p>

<div class="image"><img src="images/S-1.png" /><p>Figure 1 The safety principle for student lab WHU.</p></div>

<p>Responsibility distribution: In order to minimize potential
injuries, each member must be aware of the experimental process from
close observations. And then, tasks will be arranged to them by our
group leader who has received strict training in his own lab. More
importantly, every member records details of his/her experiments in a
notebook which will be checked regularly. Each member takes
responsibility of cleaning and checking risks in the lab. What’s more,
we have a special guy who reports regularly on safety and daily
function of our group to Miss Long Yan, who is authorized by TCEB.</p>

<p>Management of Chemicals, Equipments and Experimenters: Chemicals
and equipments are registered in TCEB before they are available for
us.</p>

<p>Training: Our team members have been trained after Guidance of
Student Experiments formulated by TCEB.</p>

</div>

</div>
</section>

<section id="notebook" class='hidden'>
<div>

<div class="timeline">
<a name="timeline"></a>
<h2>Timeline</h2>

<pre>
<p>2013 </p>
<p>Nov. - Dec.,	Team building</p>
<p></p>
<p>2014</p>
<p>Jan. - Feb.,	Knowledge preparing. Looked into iGEM and previous teams’ wiki.</p>
<p></p>
<p>Mar 			Brainstorming</p>
<p>1st,			Put forward our idea to reduce formaldehyde</p>
<p>8th,			the idea was made our project purpose. Decided to start with acid promoter</p>
<p>15th,			acid promoter plan failed. Start to focus on formaldehyde promoter.</p>
<p>19th,			first campus weekly thesis defense.</p>
<p></p>
<p>Apr.,		Idea building.</p>
<p>10th,	 		last Campus Thesis Defense.</p>
<p>20th,	internal adjustment. Established the group framework. Group members increased to 30.</p>
<p>25th,	 		with primers, managed to generate qualified promoters</p>
<p></p>
<p>May,  		Instructor assessment and suggestions.</p>
<p>3rd, 			lab cleaning. “Assembly Line” mode was introduced. Timing of preliminary stage.</p>
<p>4th,		 	the result of the identification of the single colony which was positive.</p>
<p></p>
<p>Jun. ~ Oct., 	Experiment and Human Practice</p>
<p>Jun. 30th, 	Group meeting. T-easy plasmid was introduced.</p>
<p>Aug. 15th, 	Group meeting.</p>
<p>Aug. 18th, 	Group members set off to HZAU for investigation and study.</p>
<p>Aug. 20th,		Debug of Human Practice. Team networking.</p>
<p>Aug. 26th, 	Attended CCiC held in Huazhong Agricultural University.</p>
<p>Sept. 4th, 	    Decided group Logo. First sketch of our poster. </p>
<p>Sept. 6th, 	    Got first result to prove formaldehyde promoter functional.</p>
<p>Sept. 10th, 	Mathematical Modeling preparation.</p>
<p>Sept. 20th,	    PowerPoint materials preparation.</p>
<p>Sept. 28th, 	Biobrick shipping.</p>
<p>Sept. 30th, 	Another biobrick shipping. </p>
<p>Oct. 1st, 	    Assisted HUST-Innovators in constructing a biobrick</p>
<p>Oct. 4th, 	    Prepare for mathematical modeling and establishment of website</p>
<h3>Multi-Assembly-Line Journal:</h3>
<h4>Promoter<h4>
<p>Jun. ~ Sept., 	Pre-experiment for the construction and function test of hxlR operon.</p>
<p>Aug. 20th, 	hxlR-GFP completed.</p>
<p>Sept., 		Got a passable result with a small amount of leakage.  </p>   
<h4>Lux Systerm</h4>
<p>Sept. 26th, 	Construction complete. </p>
<p>Oct. 1st, 		Function test showed a positive result</p>.
<h4>FDH, PADH</h4>
<p>Jul. ~ Aug., 	Enzyme DNA extracting and processing. </p>
<h4>E7</h4>
<p>Aug. 20th, 	Lysing function test. Western blot processing and OD600 observing.</p>

</pre>

</div>

<div class="methods">
<a name="methods"></a>
<h2>Methods</h2>

<h3>Function Test of E7 Lysis system</h3>

<h4>The construction of E7-pUHE21 contained strain w3110</h4>

<p>BBa_K117000 which contains E7 lysis gene is found out from iGEM
2014 registry and primers are design to PCR the target gene(the
sequence of the primers are shown below) BBa_B0034 is contained in the
upstream primer, which is a strong RBS.The plasmid pUHE21 is a
commercialized one that can be extracted from a modified strain of
E.coli(part of its sequence is shown below). EcoRI and PstI are used
to cut both the PCR product and the pUHE21 backbone. After
purification and ligation with T4 ligase, the reconstructed E7-pUHE21
plasmid is transformed into competent E.coli strain w3110. The
positive clone with right DNA sequence inserted is screened out and
referred to as strain E7</p>

<p>Sequence of upstream primer with strong RBS (BBa_B0034):
CGGAATTCTTCTAGAGAAAGAGGAGAAATACTAGATGAAAAAAATAACAGGGATTAT</p>

<p>Sequence of downstream primer:
AACTGCAGGCGGCCGCTACTAGTATTACTGCGTTTCCACTCCG</p>

<h4>The construction of pUHE21 contained strain w3110</h4>

<p>In order to make a negative control of this experiment, pUHE21 is
transformed into competent E.coli strain w3110 and the positive clone
is screened out, referred to as strain BB(short for backbone).</p>

<h4>IPTG induction and OD measurement</h4>

<p>Inoculate 3 Erlenmeyer flasks (referred to as E7-0, E7-1 and E7-2
separately) containing 100ml LB medium with 1ml of saturated overnight
strain E7 culture, and at the same time inoculate 2 Erlenmeyer flasks
(referred to as BB-0 and BB-1 separately) containing 100ml LB medium
with 1ml of saturated overnight strain BB culture. All of the broth
are cultured to OD=0.5, then IPTG is added to E7-1, E7-2 and BB-1 to
make their IPTG concentration reach 1.0mM, 2.0mM and 1.0mM,
respectively. After induction those 5 Erlenmeyer flasks are cultured
for another 8 hours, during which 3 samples are taken into a 96-well
plates from each flask every hours and their OD are measured with
microplate reader.</p>

<h4>Western blot to further prove the lysis effect of E7</h4>

<p>Besides OD measurement, we also do Western blot. Taken from each
Erlenmeyer flasks at 0, 1, 2, 4, 6 hours after induction, those
samples are centrifuged ,the supernatant and sediment are used to run
a SDS-PAGE, then the antibody of DnaK(which is an endogenous protein
expressed constitutively in E.coli) is used to conduct a western
blot.</p>

<h3>Function Test of The Enzymes</h3>

<p>To test whether formaldehyde dehydrogenase (PADH) work or not, we
developed this system. In this system, when PADH oxidize formaldehyde,
electrons lost by formaldehyde are transferred to nicotinamide-adenine
dinucleotide (NAD+). Gaining electrons, the NAD+ are reduced to
reduced form of nicotinamide-adenine dinucleotide (NADH). By measuring
the absorbance of NADH under light of 340nm, we can know the
concentration of NADH, and finally know the activity of PADH.</p>

<h4>Culture</h4>

<p>Experimental group (E.coli with PADH gene) and control group
(E.coli without PADH gene) are both cultured under 37&#8451, overnight</p>

<h4>Lysis</h4>

<p>Collect cells by centrifugation, and wash them with Tris-HCl (PH 7.5) twice</p>

<p>Resuspend cells with Tris-HCl (PH 7.5)</p>

<p>Lyse them by ultrasonic cell disrupter: 500w, work 5s, interval 7s,
repeat dozens of times until the solute becomes clear.</p>

<p>Collect supernatant by cryogenic centrifugal (4&#8451;, 12 000 r/min , 20min)</p>

<h4>Enzyme activity assay</h4>

<p>Prepare the reaction system in colorimetric tube: 1.5ml Tris-HCl
(PH 8.0), 0.5ml NAD+ (6mM), 0.5ml formaldehyde solution (12mM), 2.5ml
total</p>

<p>Incubate under 50&#8451; for 10min, measure the absorbance from time to
time until stabilized</p>

<p>Add 0.5ml supernatant (experimental group) or Tris-HCl buffer
(control group) to the reaction system, and continue incubating</p>

<p>Measure the absorbance under 340nm wavelength of light, after 5min,
10min, 15min, 20min respectively</p>

</div>

</div>
</section>

</main>

<script src="jquery-2.1.1.min.js"></script>
<script src="main.js"></script>

</body>
</html>
